Dockerfile-debian.template

```diff
@@ -26,6 +26,8 @@ RUN set -eux; \
 # grab tini for signal processing and zombie killing
 		tini \
 	; \
+# allow imagemagick to use ghostscript (4.1+)
+	sed -Ei 's@(<policy domain="coder" rights=)"none"( pattern="PDF"\s*/>)@\1"read"\2@' /etc/ImageMagick-6/policy.xml; \
```

Comment:
I'd like to simplify the `sed` a little to focus on the target bits.
```suggestion
# allow imagemagick to use ghostscript for PDF -> PNG thumbnail conversion (4.1+)
	sed -ri 's/(rights)="none" (pattern="PDF")/\1="read" \2/' /etc/ImageMagick-6/policy.xml; \
```

---

We'll also need to add `-e '!ImageMagick-6/policy.xml!d` to the `commonSedArgs` on line 46 of `update.sh` so that the new lines don't show up on the `4.0` Dockerfile.

https://github.com/docker-library/redmine/blob/3a41d66499222d90f2f4207c7f6dafd38f4001c5/update.sh#L46-L48